aspnetcore/fundamentals/servers/kestrel/endpoints.md

```diff
@@ -199,7 +200,24 @@ In the following *appsettings.json* example:
 }
 ```
 
-An alternative to using `Path` and `Password` for any certificate node is to specify the certificate using certificate store fields. For example, the `Certificates` > `Default` certificate can be specified as:
+Schema notes:
+
+* Endpoints names are [case-insensitive](xref:fundamentals/configuration#configuration-keys-and-values). For example, `HTTPS` and `Https` are valid.
+* The `Url` parameter is required for each endpoint. The format for this parameter is the same as the top-level `Urls` configuration parameter except that it's limited to a single value.
+* These endpoints replace those defined in the top-level `Urls` configuration rather than adding to them. Endpoints defined in code via `Listen` are cumulative with the endpoints defined in the configuration section.
```

Comment:
```suggestion
* These endpoints replace those defined in the top-level `Url` configuration rather than adding to them. Endpoints defined in code via `Listen` are cumulative with the endpoints defined in the configuration section.
```
Is `Urls` the plural of `Url`? That's typically written as `Url`s which is bad, but not as bad as `Urls`.

Reply:
I believe there is a setting called `Urls` that this refers to.

Reply:
> I believe there is a setting called `Urls` that this refers to.

Can you confirm that?
It's not in the sample file, but that's OK.

Reply:
It refers to the Urls mentioned at the start of the article - https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel/endpoints

That part should be split out into its own section so others can link to it.